pkg/namesgenerator/names-generator.go

```diff
@@ -830,6 +830,9 @@ var (
 
 		// Nikolay Yegorovich Zhukovsky (Russian: Никола́й Его́рович Жуко́вский, January 17 1847 – March 17, 1921) was a Russian scientist, mathematician and engineer, and a founding father of modern aero- and hydrodynamics. Whereas contemporary scientists scoffed at the idea of human flight, Zhukovsky was the first to undertake the study of airflow. He is often called the Father of Russian Aviation. https://en.wikipedia.org/wiki/Nikolay_Yegorovich_Zhukovsky
 		"zhukovsky",
+
+		// Satoshi Nakamoto is the name used by the unknown person or group of people who developed bitcoin, authored the bitcoin white paper, and created and deployed bitcoin's original reference implementation. https://en.wikipedia.org/wiki/Satoshi_Nakamoto
+		"satoshi",
```

Comment:
Satoshi is his first name, not last name

Also, please add a female as well: https://github.com/moby/moby/blob/05836c222d4f3dc073846422087d75953a84f283/pkg/namesgenerator/names-generator.go#L122
(I'm not sure Satoshi's real gender, but his/her name looks like male)

Reply:
Hi @AkihiroSuda, thanks for the prompt response.

I've used the first name because it is the most common name used when referring to the Bitcoin creator. Also, I didn't add another person because no one knows if Satoshi is a woman or a man and adding another female person would imply Satoshi is a man (and I would prefer to avoid making this conclusion).